tools/rum-models-generator/Sources/RUMModelsGeneratorCore/Input/ObjcInterop/SwiftToObjcInteropTypeTransformer.swift

```diff
@@ -135,13 +65,105 @@ internal class SwiftToObjcInteropTypeTransformer {
             return ObjcInteropNSArray(element: try objcInteropType(for: swiftArray.element))
+        case let swiftDictionary as SwiftDictionary:
+            return ObjcInteropNSDictionary(
+                key: try objcInteropType(for: swiftDictionary.key),
+                value: try objcInteropType(for: swiftDictionary.value)
+            )
         default:
             throw Exception.unimplemented(
                 "Cannot create `ObjcInteropType` type for \(type(of: swiftType))."
             )
         }
     }
 
+    private func objcInteropPropertyWrapper(for swiftProperty: SwiftStruct.Property, in objcClass: ObjcInteropClass) throws -> ObjcInteropPropertyWrapper {
+        switch swiftProperty.type {
+        case let swiftPrimitive as SwiftPrimitiveType:
+            let propertyWrapper = ObjcInteropPropertyWrapperManagingSwiftStructProperty(
+                owner: objcClass,
+                swiftProperty: swiftProperty
+            )
+            propertyWrapper.objcInteropType = try objcInteropType(for: swiftPrimitive)
+            return propertyWrapper
+        case let swiftStruct as SwiftStruct:
+            let propertyWrapper = ObjcInteropPropertyWrapperAccessingNestedStruct(
+                owner: objcClass,
+                swiftProperty: swiftProperty
+            )
+            propertyWrapper.objcNestedClass = ObjcInteropTransitiveClass(
+                owner: propertyWrapper,
+                bridgedSwiftStruct: swiftStruct
+            )
+            return propertyWrapper
+        case let swiftEnum as SwiftEnum:
+            let propertyWrapper = ObjcInteropPropertyWrapperAccessingNestedEnum(
+                owner: objcClass,
+                swiftProperty: swiftProperty
+            )
+            propertyWrapper.objcNestedEnum = ObjcInteropEnum(
+                owner: propertyWrapper,
+                bridgedSwiftEnum: swiftEnum
+            )
+            return propertyWrapper
+        case let swiftArray as SwiftArray where swiftArray.element is SwiftEnum:
+            let propertyWrapper = ObjcInteropPropertyWrapperAccessingNestedEnumsArray(
+                owner: objcClass,
+                swiftProperty: swiftProperty
+            )
+            propertyWrapper.objcNestedEnumsArray = ObjcInteropEnumArray(
+                owner: propertyWrapper,
+                bridgedSwiftEnum: swiftArray.element as! SwiftEnum // swiftlint:disable:this force_cast
+            )
+            return propertyWrapper
+        case let swiftArray as SwiftArray where swiftArray.element is SwiftPrimitiveType:
+            let propertyWrapper = ObjcInteropPropertyWrapperManagingSwiftStructProperty(
+                owner: objcClass,
+                swiftProperty: swiftProperty
+            )
+            propertyWrapper.objcInteropType = try objcInteropType(for: swiftArray)
+            return propertyWrapper
+        case let swiftDictionary as SwiftDictionary where swiftDictionary.value is SwiftPrimitiveType:
```

Comment:
The whole function comes from the block of code previously starting at line 35. But I want to draw attention to this case that handles a `SwiftDictionary`.